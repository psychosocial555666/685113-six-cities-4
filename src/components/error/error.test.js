import Error from "./error.jsx";
import React from "react";
import renderer from "react-test-renderer";

it(`Should Error render correctly`, () => {
  const tree = renderer
      .create(<Error />,
          {createNodeMock: () => {
            return document.createElement(`div`);
          }})
      .toJSON();

  expect(tree).toMatchSnapshot();
});
