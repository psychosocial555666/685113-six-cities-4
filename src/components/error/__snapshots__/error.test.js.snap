// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Error render correctly 1`] = `
<div
  style={
    Object {
      "backgroundColor": "red",
      "boxShadow": "0 14px 28px red",
      "color": "white",
      "fontSize": "20px",
      "fontWeight": "bold",
      "position": "fixed",
      "right": 0,
      "textAlign": "center",
      "top": 0,
      "width": "100%",
    }
  }
>
  No comment sent, check your internet connection
</div>
`;
